From 03626d7940d5e2fc2abf65ef1b49a7a89623f08f Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Sun, 14 Sep 2014 13:20:29 +0200
Subject: [PATCH] fdo#80650 src transl. merge must be case sensitive

This makes the desired changes in
workdir/SrsPartMergeTarget/starmath/source/symbol.src. The only other
merged file that is changed is
workdir/SrsPartMergeTarget/sw/source/ui/misc/numberingtypelistbox.src,
with changes like

                < "1, 2, 3, ..." ; 4/*SVX_NUM_ARABIC
-               < "a, b, c, ..." ; 0/*SVX_NUM_CHARS_UPPER_LETTER   */; > ;
+               < "A, B, C, ..." ; 0/*SVX_NUM_CHARS_UPPER_LETTER   */; > ;
                < "a, b, c, ..." ; 1/*SVX_NUM_CHARS_LOWER_LETTER   */; > ;

This looks okay to me too :-)

Change-Id: I729075209027ed1f3fec311c05b631c0f681708b
---
 l10ntools/source/export.cxx | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/l10ntools/source/export.cxx b/l10ntools/source/export.cxx
index 49805cc..6052e24 100644
--- a/l10ntools/source/export.cxx
+++ b/l10ntools/source/export.cxx
@@ -275,7 +275,7 @@ Export::~Export()
 
     if ( bMergeMode ) {
         if ( !pMergeDataFile )
-            pMergeDataFile = new MergeDataFile(sMergeSrc, global::inputPathname, false);
+            pMergeDataFile = new MergeDataFile(sMergeSrc, global::inputPathname, true);
 
         delete pMergeDataFile;
     }
@@ -1062,14 +1062,14 @@ void Export::ResData2Output( MergeEntrys *pEntry, sal_uInt16 nType, const OStrin
 void Export::MergeRest( ResData *pResData )
 {
     if ( !pMergeDataFile ){
-        pMergeDataFile = new MergeDataFile( sMergeSrc, global::inputPathname, false );
+        pMergeDataFile = new MergeDataFile( sMergeSrc, global::inputPathname, true );
         aLanguages = pMergeDataFile->GetLanguages();
 
     }
 
     MergeEntrys *pEntry = 0;
     if( pResData->bText || pResData->bQuickHelpText || pResData->bTitle )
-        pEntry = pMergeDataFile->GetMergeEntrys( pResData );
+        pEntry = pMergeDataFile->GetMergeEntrysCaseSensitive( pResData );
 
     if ( pEntry )
     {
@@ -1154,7 +1154,7 @@ void Export::MergeRest( ResData *pResData )
                     ConvertExportContent( pResData->sId );
                 }
 
-                MergeEntrys* pEntrys = pMergeDataFile->GetMergeEntrys( pResData );
+                MergeEntrys* pEntrys = pMergeDataFile->GetMergeEntrysCaseSensitive( pResData );
                 OString sText;
                 bool bText = pEntrys && pEntrys->GetText( sText, STRING_TYP_TEXT, sCur, true );
 
-- 
1.9.3

